src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -17,7 +17,7 @@
  */
 class Tribe__Tickets__Shortcodes__User_Event_Confirmation_List {
 	protected $shortcode_name = 'tribe-user-event-confirmations';
-	protected $params = array();
+	protected $params = [];
```

Comment:
* :no_entry_sign: Missing member variable doc comment
* :warning: Equals sign not aligned with surrounding assignments; expected 9 spaces but found 1 space

<sub>hash: feb07569085661ced97c1dceca59a832</sub>